packages/boxel-ui/addon/src/components/add-button/index.gts

```diff
@@ -39,15 +41,21 @@ const AddButton: TemplateOnlyComponent<Signature> = <template>
       {{yield}}
     </button>
   {{else}}
-    <IconButton
-      @icon={{PlusCircleIcon}}
-      @width='40px'
-      @height='40px'
-      class='add-button'
-      aria-label='Add'
-      data-test-create-new-card-button
-      ...attributes
-    />
+    {{#if (bool @loading)}}
+      <button class='add-button loading' disabled ...attributes>
```

Comment:
Since this button is always disabled maybe it should not be a button but a div

Reply:
Good point @jurgenwerk , I changed to div instead. Thanks!